bambi/plots/plot_types.py

```diff
@@ -45,8 +45,8 @@ def plot_numeric(
     if len(covariates) == 1:
         ax = axes[0]
         values_main = transform_main(plot_data[main])
-        ax.plot(values_main, y_hat_mean, solid_capstyle="butt", color="C0")
-        ax.fill_between(values_main, y_hat_bounds[0], y_hat_bounds[1], color="C0", alpha=0.4)
+        ax.plot(values_main, y_hat_mean, solid_capstyle="butt")
```

Comment:
The `color="C0"` was added in #689 to fix #685. We should keep the fix here as well (unless you have a better option, of course)

Reply:
Definitely. I must have neglected this in my rebase when I was resolving conflicts.